packages/genesys-spark-components/src/components/stable/gux-button-multi/tests/gux-button-multi.e2e.ts

```diff
@@ -2,14 +2,7 @@ import { E2EPage } from '@stencil/core/testing';
 
 import { newSparkE2EPage, a11yCheck } from '../../../../test/e2eTestUtils';
 
-const axeExclusions = [
-  {
-    issueId: 'aria-required-children',
-    target: 'gux-button-multi,gux-list',
-    exclusionReason:
-      'To be addressed in COMUI-2387. New violation picked up after upgrading from axe-core v4.4.2 to v4.8.2'
-  }
-];
+const axeExclusions = [];
```

Comment:
what changed to make this (and other) axeExclusions no longer necessary?

Reply:
All the exclusions related to `aria-required-children` in `gux-list` did not need any change. Something must have changed since the exclusion that was added.